### DISCUSSION TITLE
What is XChart Gauge.cs API.

### DISCUSSION THREAD

**Discussion by 410zlz** [XCharts-Team/XCharts#243] [Q&A] [answered]
I looked for a lot of websites but couldn't find them.
Want to know how to change Gauge maximum.
How do colors change into gradients.
Thank you！

**Comment by monitor1394** [comment, marked as answer]
2.0 version:
https://github.com/XCharts-Team/XCharts/blob/2.0/Assets/XCharts/Documentation/XCharts配置项手册.md#Serie-Gauge

3.0 version:
https://xcharts-team.github.io/docs/gauge#serie
